packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/registries/registries.html

```diff
@@ -0,0 +1,207 @@
+<div class="mt-3">
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+    <h3 class="mb-3" data-translate="pci_ai_dashboard_registries_title"></h3>
+    <p data-translate="pci_ai_dashboard_registries_description"></p>
+
+    <h4
+        class="mb-1"
+        data-translate="pci_ai_dashboard_private_docker_registries_title"
+    ></h4>
+    <p
+        data-translate="pci_ai_dashboard_private_docker_registries_description_1"
+    ></p>
+    <p
+        data-translate="pci_ai_dashboard_private_docker_registries_description_2"
+    ></p>
+    <oui-datagrid
+        id="publicCloudTrainingRegistrysDatagrid"
+        data-rows="$ctrl.registryList"
+    >
+        <oui-datagrid-column
+            data-title=":: 'pci_pai_dashboard_private_docker_registries_id' | translate"
+            data-property="id"
+            data-type="string"
+            data-searchable
+            data-filterable
+            data-sortable
+        >
+            <a data-ng-href="{{:: $ctrl.jobInfoLink($value) }}" target="_top">
+                <span data-ng-bind="::$value"></span>
+            </a>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_ai_dashboard_private_docker_registries_region' | translate"
+            data-property="region"
+            data-type="string"
+            data-searchable
+            data-filterable
+            data-sortable
+        >
+            <span data-ng-bind="$value"></span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_ai_dashboard_private_docker_registries_url' | translate"
+            data-property="url"
+            data-type="string"
+            data-searchable
+            data-filterable
+            data-sortable
+        >
+            <span data-ng-bind="$value"></span>
```

Comment:
**nitpick:** Unecessary `span`. The result will be same even without this `span` as `oui-datagrid-column` will bind `property` attribute.
```suggestion
```